guides/common/modules/proc_configuring-project-for-performance.adoc

```diff
@@ -3,3 +3,5 @@
 
 {Project} comes with a number of components that communicate with each other.
 You can tune these components independently of each other to achieve the maximum possible performance for your scenario.
+
+We have not seen significant performance difference between {Project} installed on {EL} 7 and {EL} 8.
```

Comment:
A further note: to avoid `we` there is `{Team}`. It should be noted that in some cases it is singular, plural in others:
```console
$ rg team guides/common/attributes* -i
guides/common/attributes-satellite.adoc
89::Team: Red{nbsp}Hat

guides/common/attributes-orcharhino.adoc
24::Team: ATIX AG

guides/common/attributes-base.adoc
100::Team: Foreman developers
```